### ISSUE TITLE
Swift Package Manager

### ISSUE THREAD

**Issue by MLADevelopmentTeam** [datatheorem/TrustKit#215]
May be more of a timing thing rather than an issue, but the latest tags do not include a package manifest (Package.swift) in them.  To use with Swift Package Manager you have to specify the master branch instead of version.

**Comment by nabla-c0d3** [comment]
I will add a version, but can you confirm that the current master branch (I just pushed a few changes) does work fine with the Swift package manager? Thanks!

**Comment by tadija** [comment]
@nabla-c0d3 I can confirm that it works with SwiftPM when package is declared with the branch:

```
.package(
    url: "https://github.com/datatheorem/TrustKit.git",
    .branch("master")
),
```

**Comment by nabla-c0d3** [comment]
I released version 1.6.4 just now. @tadija thanks for checking - can you confirm that v1.6.4 works too? Thanks!

**Comment by tadija** [comment]
Hi @nabla-c0d3, it seems that v1.6.4 doesn't work with SwiftPM:

<img width="390" alt="Screen Shot 2019-12-27 at 10 52 01" src="https://user-images.githubusercontent.com/2762374/71512733-d069f880-2897-11ea-89dc-69c9449dd277.png">

This is the revision which works (from my Package.resolved):

```
{
  "package": "TrustKit",
  "repositoryURL": "https://github.com/datatheorem/TrustKit.git",
  "state": {
    "branch": "master",
    "revision": "df3e13732604396ff537d84717346b1b7e794141",
    "version": null
  }
}
```

**Comment by kunalverma25** [comment]
Same here.
Used the commit id "df3e13732604396ff537d84717346b1b7e794141" to make it work.

**Comment by jttoddy** [comment]
Using the Swift Package Manager within Xcode 11.3 with the above mentioned details from @tadija still won't build correctly after importing, saying that "Umbrella directory `[redacted]/DerivedData/SnapSendSolve/SourcePackages/checkouts/TrustKit/TrustKit/include` not found". I've also tried using latest master and the v1.6.4 release branch. 😞 Falling back on CocoaPods for now though 😌

**Comment by piobyz** [comment]
I was able to convert TrustKit from Carthage to SPM in my project, but only with an exact commit df3e13732604396ff537d84717346b1b7e794141 @tadija and @kunalverma25 mentioned above.

SPM pointing to v1.6.4 caused the same errors as above.

I also needed to remove #import <TrustKit/TrustKit.h> from my ...Bridging-Header.h, but seems like it did no harm.

I'm using Xcode 11.3.1.

@nabla-c0d3 Looks like problem was introduced with commit 71a91932788d8931b46a4c37f50e7e15d17ea523 "Fix location of TrustKit.h for CocoaPods; fixes #216", library compiled with SPM still looks for TrustKit.h in old location.
